build.gradle

```diff
@@ -38,26 +37,16 @@ subprojects {
   }
 
   dependencies {
-    compile 'com.amazonaws:aws-java-sdk:1.11.18'
-    compile 'com.fasterxml.jackson.core:jackson-databind:2.5.5'
+    compile 'com.amazonaws:aws-java-sdk:1.11.28'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.6.6'
```

Comment:
this is the jackson version that the aws-java-sdk is using. previous release was getting updated to this anyway but this makes it explicit